dateutil/parser.py

```diff
@@ -35,7 +35,7 @@
 import collections
 import re
 from io import StringIO
-from calendar import monthrange, isleap
```

Comment:
The last commit that this was needed in was ad8c744, removed in 1622e2b4c596. This can be removed.